js/Player.js

```diff
@@ -0,0 +1,26 @@
+class Player {
+  
+  constructor(name) {
+    this.name = name;
+    this.points = 0;
+  };
+  
+  submitAnswer(userGuess, clue) {
+    if (userGuess === clue.answer) {
+      this.points += clue.pointValue
+    } else {
+      this.points -= clue.pointValue
+    };
+  };
+}
+
+if (module !== 'undefined') {
```

Comment:
I believe this should be `if (typeof module !== 'undefined')`

Reply:
That's correct ^^ you'll want to make sure you are using the `typeof` operator.